### ISSUE TITLE
Fail to build Gitbook after recent update

### ISSUE THREAD

**Issue by drhoffmann** [zanfab/gitbook-plugin-intopic-toc#15]
Hi

After reinstalling v1.1.0 I get an error when running `gitbook build`

```
info: loading plugin "intopic-toc"... ERROR

PluginError: Error with plugin "intopic-toc": Cannot find module './dist/index.js'
```

**Comment by fzankl** [comment]
Should be fixed with release 1.1.1
Please try again using the new version and feel free to open a new issue if any other problems occur.

**Comment by drhoffmann** [comment]
Works like a charm. Thank you for the very quick fix
